db/migrate/20230316034707_disable_inherits_tax_category_from_per_order_enterprise_fees.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class DisableInheritsTaxCategoryFromPerOrderEnterpriseFees < ActiveRecord::Migration[6.1]
+  class EnterpriseFee < ApplicationRecord
+  end
+
+  class Spree::Calculator < ApplicationRecord
+  end
+
+  def change
+    EnterpriseFee.where(id: enterpise_fee_ids)
+      .where(inherits_tax_category: true)
+      .update_all(inherits_tax_category: false)
+  end
+
+  def enterpise_fee_ids
+    Spree::Calculator.where(
+      calculable_type: 'EnterpriseFee',
+      type: per_order_calculators
+    ).pluck(:calculable_id)
+  end
```

Comment:
Not that efficiency matters here, I thought it worth pointing out that we could probably use a `join` to avoid the extra query.

Reply:
Yes, or at least a `select` to have a subquery.

Reply:
I replaced the old code with a join